sdk/go/common/workspace/project.go

```diff
@@ -93,7 +93,13 @@ type ProjectTemplateConfigValue struct {
 	Secret bool `json:"secret,omitempty" yaml:"secret,omitempty"`
 }
 
-// ProjectBackend is a configuration for backend used by project
+// Configure which state backend for the project to use. If unset, will use the
```

Comment:
Documentation style is we always start the doc comment with the name of thing its describing, so this should be something like "ProjectBackend is the configuration for which state backend...."